R/install-remote.R

```diff
@@ -17,12 +17,12 @@ install_remote <- function(remote, ..., quiet = FALSE) {
   source <- source_pkg(bundle, subdir = remote$subdir)
   on.exit(unlink(source, recursive = TRUE), add = TRUE)
 
-  add_metadata(source, remote_metadata(remote, bundle, source))
+  metadata <- remote_metadata(remote, bundle, source)
 
   # Because we've modified DESCRIPTION, its original MD5 value is wrong
   clear_description_md5(source)
```

Comment:
so remove the `clear_description_md5`, as it is no longer needed because the metadata is not being added **before** install?